@navikt/core/react/src/form/combobox/combobox.stories.tsx

```diff
@@ -186,8 +185,6 @@ export const WithAddNewOptions: StoryFn = ({ open }: { open?: boolean }) => {
       options={options}
       allowNewValues={true}
       shouldAutocomplete={true}
-      value={value}
-      onChange={setValue}
```

Comment:
I think removal of `value` here and in MultiSelectWithAddNewOptions "breaks" the Chromatic story.